torch/nn/functional.py

```diff
@@ -1651,19 +1652,32 @@ def rrelu(
 )
 
 
-def gelu(input):
-    r"""gelu(input) -> Tensor
+def gelu(input: Tensor, approximate: str = 'none') -> Tensor:
```

Comment:
Please make this new arg kw-only: `def gelu(input: Tensor, *, approximate: str = 'none') -> Tensor`. Similar changes may be needed throughout

Reply:
I'm seeing this torchscript error:
```
torch.jit.frontend.NotSupportedError: Compiled functions can't take variable number of arguments or use keyword-only arguments with defaults:
  File "/opt/conda/lib/python3.7/site-packages/torch/nn/functional.py", line 1655
def gelu(input: Tensor, *, approximate: str = 'none') -> Tensor:
                                              ~~~~~ <--- HERE
    r"""gelu(input, approximate = 'none') -> Tensor
```

Reply:
That's no good. We'll have to leave this as non-kw-only, sadly.

Reply:
We definitely should!